src/lists.c

```diff
@@ -2807,6 +2807,38 @@ static Int CheckInit (
     /* check implications                                                  */
     for ( i = FIRST_LIST_TNUM;  i <= LAST_LIST_TNUM;  i++ ) {
 
+        if ( (i & IMMUTABLE) == 0 ) {
+            if ( ClearFiltsTNums[i]+IMMUTABLE != ClearFiltsTNums[i+IMMUTABLE]) {
+                Pr( "#W  ClearFiltsTNums [%s] mismatch between mutable and immutable\n",
+                    (Int)(InfoBags[i].name), 0 );
+                success = 0;
+            }
+            for ( j = 0;  j < ARRAY_SIZE(fnums);  j++ ) {
```

Comment:
To be clear: that was a serious open question, not sarcasm: Looking at `CheckInit`, it is very long and non-trivial to read, and could benefit from some refactoring, though I punted on that fo now, in order to not get distracted :-)